zerver/views/webhooks/stripe.py

```diff
@@ -0,0 +1,129 @@
+# Webhooks for external integrations.
+from __future__ import absolute_import
+from django.utils.translation import ugettext as _
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.lib.validator import check_dict, check_string
+from zerver.models import Client, UserProfile
+
+from django.http import HttpRequest, HttpResponse
+from six import text_type
+from typing import Dict, Any, Iterable, Optional
+
+from datetime import datetime
+import decimal
+
+from past.utils import old_div
+
+@api_key_only_webhook_view('Stripe')
+@has_request_variables
+def api_stripe_webhook(request, user_profile, client,
+                       payload=REQ(argument_type='body'), stream=REQ(default='test'),
+                       topic=REQ(default='Stripe')):
+    # type: (HttpRequest, UserProfile, Client, Dict[str, Any], text_type, Optional[text_type]) -> HttpResponse
+    body = ""
+    event_type = ""
+    try:
+        event_type = "{type}".format(**payload)
```

Comment:
This is sorta a confusing way to write this; why not `event_type = payload["type"]`?